lib/registerPlugins.js

```diff
@@ -1,21 +1,64 @@
 'use strict';
 const async = require('async');
 
-module.exports = (server, callback) => {
+/**
+ * Register the default Screwdriver API plugins
+ * @method registerDefaultPlugins
+ * @param  {Object}   server        Hapi server object
+ * @param  {Function} callback
+ */
+function registerDefaultPlugins(server, callback) {
     const plugins = [
         'inert',
         'vision',
         '../plugins/status',
         '../plugins/logging',
         '../plugins/swagger'
-    ].map((plugin) => require(plugin)); // eslint-disable-line global-require
+    /* eslint-disable global-require */
+    ].map((plugin) => require(plugin));
+    /* eslint-enable global-require */
 
-    // TODO: possible performance boost on server startup by only calling register once
```

Comment:
Yeah that would be a better place to track it. I didn't want to put TODO stuff in code.